packages/io-ts-http/src/combinators.ts

```diff
@@ -13,10 +14,51 @@ import {
   Simplify,
 } from './utils';
 
+const partialWithoutUndefined = <P extends t.Props>(
+  props: P,
+  name?: string,
+): t.PartialType<
+  P,
+  DefinedValues<t.TypeOfPartialProps<P>>,
+  DefinedValues<t.OutputOfPartialProps<P>>
+> => {
+  const partialCodec = t.partial(props, name);
+  return new t.PartialType(
+    partialCodec.name,
+    (i): i is DefinedValues<t.TypeOfPartialProps<P>> => partialCodec.is(i),
+    (i, ctx) => {
+      return pipe(
+        partialCodec.validate(i, ctx),
+        E.map((result) => {
+          for (const key in result) {
```

Comment:
Should we use `Object.keys` here, which only enumerates owned properties?

```suggestion
          for (const key of Object.keys(result)) {
```

Reply:
Similar to this change: https://github.com/BitGo/api-ts/pull/277